packages/frontend/src/pages/scaling/costs/props/getScalingCostsColumnsConfig.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  TypeCell,
+  TypeColumnTooltip,
+} from '../../../../components/table/TypeCell'
+import { ColumnConfig, SortingConfig } from '../../../../components/table/types'
+import { cn } from '../../../../utils/cn'
+import { CostsDataDetails, ScalingCostsViewEntry } from '../types'
+import { CostsTableCell } from '../view/CostsTimeRangeCell'
+
+export function getScalingCostsColumnsConfig() {
+  const columns: ColumnConfig<ScalingCostsViewEntry>[] = [
+    ...getProjectWithIndexColumns({ indexAsDefaultSort: true }),
+    {
+      name: 'Type',
+      tooltip: <TypeColumnTooltip />,
+      shortName: 'Type',
+      getValue: (project) => (
+        <TypeCell provider={project.provider}>{project.category}</TypeCell>
+      ),
+      sorting: {
+        getOrderValue: (project) => project.category,
+        rule: 'alphabetical',
+      },
+    },
+    {
+      type: 'group',
+      columns: [
+        {
+          name: 'Total Cost',
+          getValue: (project) => (
+            <CostsTableCell data={project.costs} type="total" />
+          ),
+          tooltip:
+            'The sum of the costs for calldata, blob data, computation, and an additional 21000 gas overhead per transaction for the selected time period.',
+          align: 'center',
+          sorting: getSorting('total'),
+        },
+      ],
+    },
+    {
+      name: 'Calldata',
+      getValue: (project) => (
+        <CostsTableCell data={project.costs} type="calldata" />
+      ),
+      headClassName: underlineClassNames(
+        'before:bg-blue-700',
+        'dark:before:bg-blue-400',
+      ),
+      tooltip:
+        'The sum of the costs for posting data as calldata on Ethereum for the selected time period.',
+      align: 'right',
+      sorting: getSorting('calldata'),
+    },
+    {
+      name: 'Blobs',
+      getValue: (project) => (
+        <CostsTableCell data={project.costs} type="blobs" />
+      ),
+      headClassName: underlineClassNames(
+        'before:bg-orange-400',
+        'dark:before:bg-yellow-100',
+      ),
+      tooltip:
+        'The sum of the costs for posting data as blobs on Ethereum for the selected time period.',
+      align: 'right',
+      sorting: getSorting('blobs'),
+    },
+    {
+      name: 'Compute',
+      getValue: (project) => (
+        <CostsTableCell data={project.costs} type="compute" />
+      ),
+      headClassName: underlineClassNames('before:bg-pink-100'),
+      tooltip:
+        'The sum of the costs for carrying out different operations within a transaction for the selected time period.',
+      align: 'right',
+      sorting: getSorting('compute'),
+    },
+    {
+      name: 'Overhead',
+      getValue: (project) => (
+        <CostsTableCell data={project.costs} type="overhead" className="pr-4" />
+      ),
+      headClassName: underlineClassNames('!pr-4', 'before:bg-green-500'),
+      tooltip:
+        'The sum of the fixed 21000 GAS overhead per transaction for the selected time period.',
```

Comment:
Maybe we should change it for this to make it more readable
```suggestion
        'The sum of the fixed 21,000 GAS overhead per transaction for the selected time period.',
```